script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -5,190 +5,190 @@ My body temperature drops.
 <Page7>
-"―――But, I can't."//text overflow, needs rephrasing
-A person who couldn't do anything. Me, who chose to do nothing.
-A person who wasn't even armed. Me, who carried a weapon.
-Common sense and irrationality flail about within my pressurized brain.
-My consciousness turns to my rear, telling me to run. There's nothing I can do to stop that.
-It's no use. What can she do against that thing? That woman. This is more than enough. Anything more would be overkill. That woman. All she has to do is run away. Just turn away. That's what any decent person would do. That woman is going to be killed.//ruby dots <ま|・><っ|・><と|・><う|・><な|・><人|・><間|・>
+"―――But, I can't."//Do not alter this page without playtesting
+She, who could do nothing. Me, who chose to do nothing.//Crushed to fit on a single line/free up space
+She, who wasn't even armed. Me, who carried a weapon.//Crushed to fit on a single line/free up space
+Common sense and irrationality writhe within my pressurized brain.
+My consciousness turns to my rear, telling me to run. There's nothing I can do to stop him.//Is he unable to stop himself? Or stop Vlov? 逃げようと意識が背後を向く。止めようのない自分がいる。
+It's no use after all. What could I possibly do against an opponent like that? That woman. From here on I'm totally out of my depth. That woman. I should just run away. Just look away. Any decent human would. That woman, will be killed.//Ruby <ま|・><っ|・><と|・><う|・><な|・><人|・><間|・>
 <Page8>
 I look up at the sky. Frozen buildings surround my vision.
 The people that had passed away in an instant. They probably didn't even have time to scream.
 The scene at the hotel last night. I think back to the pleas of those people who were killed without reason.
-The body of the fallen woman. The corpses drool as they rub against the <woman|f l e s h   >. The expanding river of ice. Throughout, the vampire continues to observe the death of the city with indifference―――
+The fallen figure. The corpses drool sloppily as they draw near the <woman|f l e s h   >. The expanding river of ice. Throughout, the vampire continues to observe the death of the city with indifference―――//First woman dropped 倒れた女の姿。だらしなく<涎|よだれ>をたらして<女|にく>にすり寄る死体。
 <Page9>
 "――――――There's no doubt about it."
-My skull cracks. In this low temperature, my heart pumps out blood in a raging fit.
+My skull cracks. Under these freezing conditions, my heart pumps blood ferociously.
 That is, in other words...
 <Page10>
 "――――――I cannot allow this."
-The fear running down my spine turns entirely into anger.
-I'm a defect. A laughingstock. Too inadequate to be a human.# After all.# Not a single atom in my body can tolerate just observing―――!
+All of the fear running down my spine transforms into anger.
+I'm defective. A laughingstock. Too inadequate to be a human.# After all,# not a single atom in my body can tolerate just observing―――!//Inadequate seems a bit forceful for 人間としてあまりにも未熟すぎる?
```

Comment:
Seems like a mis TL. 
'Too immature as a human' is more accurate. Probably needs rewording.